06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(file):
+    data = {}
+    print
+    with open(file, "r") as csv_file:
+        try:
+            csv_reader = csv.reader(csv_file)
+            for name, score in csv_reader:
+                score = int(score)
+                if name not in data:
+                    data[name] = score
+                else:
+                    if score > data[name]:
+                        data[name] = score
+
+        except ValueError:
```

Comment:
Why ValueError? (I think it should be `csv.Error` as written in the csv module docs: https://docs.python.org/2/library/csv.html#csv.Error)